### PR TITLE
Added surface area constraint function to pyGeo MPhys wrapper

### PR DESCRIPTION
**Pull request by eytanadler**
## Purpose

There was no surface area constraint function in the MPhys wrapper for DVCon. Now there is.

## Expected time until merged

A few days

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing

When setting up the geometric constraints in an MPhys problem, call something like `self.geometry.nom_addSurfaceAreaConstraint("surface_area")` to add a geometry output that computes the surface area of the constraint surface. It works in my scripts.

## Checklist

- [ ] I have run `flake8` and `black` to make sure the Python code adheres to PEP-8 and is consistently formatted
- [ ] I have formatted the Fortran code with `fprettify` or C/C++ code with `clang-format` as applicable
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: pygeo/mphys/mphys_dvgeo.py [hunk 1]
```diff
@@ -589,6 +589,25 @@ def nom_addVolumeConstraint(self, name, leList, teList, nSpan=10, nChord=10, sca
         )
         self.add_output(name, distributed=False, val=1.0)
 
+    def nom_addSurfaceAreaConstraint(self, name, scaled=True, surface_name="default"):
```
Comment (hajdik) [inline pygeo/mphys/mphys_dvgeo.py:592]: Can you add the rest of the parameters besides bounds and scale?

**Review by hajdik — CHANGES_REQUESTED** [review]
Looks good, just a minor comment

File: pygeo/mphys/mphys_dvgeo.py (hunk 1 above)
Comment (eytanadler) [reply pygeo/mphys/mphys_dvgeo.py]: Done, added them to `addVolumeConstraint` and `addProjectedAreaCosntraint` too

**Review by hajdik — APPROVED** [review]
LGTM

**Review by hajdik — DISMISSED** [review]

**Review by hajdik — APPROVED** [review]
still LGTM

**Review by sseraj — APPROVED** [review]

**Pull request merged**